scripts/globals/abilities/afflatus_misery.lua

```diff
@@ -15,9 +15,7 @@ ability_object.onAbilityCheck = function(player, target, ability)
 end
 
 ability_object.onUseAbility = function(player, target, ability)
-    target:delStatusEffect(xi.effect.AFFLATUS_SOLACE)
-    target:delStatusEffect(xi.effect.AFFLATUS_MISERY)
-    target:addStatusEffect(xi.effect.AFFLATUS_MISERY, 8, 0, 7200)
+    xi.job_utils.white_mage.useAfflatusMisery(player, target, ability)
```

Comment:
No missing returns?

Reply:
I think I got them all this time! I tested Benediction, Devotion, and Martyr and was like wait, why are those returning 0's? Aaaah, crap, bet MNK was the same way. Yup!

Reply:
I did find that Asylum is set to only affect the using WHM when it should be party AoE. Looks like the database's validTarget just needs to be updated from 1 to 4 for party members. Should I go ahead and throw that into this PR as well?

Reply:
Sure, go for it